dev/tests/static/framework/tests/unit/testsuite/Magento/TestFramework/Utility/PartialNamespace/Baz.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\TestFramework\Utility\PartialNamespace;
```

Comment:
Could you move test files that are used in tests into the `_files` directory?

Reply:
They will not be loaded with autoloader then and we will be forced to include them manually.